lib/ui/screen/register_screen.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:covert_html_to_pdf/blocs/providers/auth_notifier.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class RegisterScreen extends ConsumerStatefulWidget {
+  const RegisterScreen({super.key});
+
+  @override
+  ConsumerState<RegisterScreen> createState() => _RegisterScreenState();
+}
+
+class _RegisterScreenState extends ConsumerState<RegisterScreen> {
+
+  final nameController = TextEditingController();
+  final emailController = TextEditingController();
+  final passwordController = TextEditingController();
+  final phoneController = TextEditingController();
```

Comment:
dispose all controller in the dispose method , or for better to remove the controller here and use formKey instead